src/index.html

```diff
@@ -11,7 +11,164 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+    
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+      <fieldset class="block">
+        <legend>Personal information</legend>
+
+        <label class="form-field input-margin">
+          Surname:
+          <input 
+          type="text" 
+          name="surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="surname"
            autocomplete="off"
          >
```
check indentations. fix all such cases

Reply:
Thank you) What should I do next after fixing the code style? Again npm i or npm start? What should my actions must be?

Reply:
Done. Check please